.github/workflows/network-operator-docker-and-helm-ci.yaml

```diff
@@ -5,26 +5,31 @@ on:
     branches:
     - "master"
     - "v[0-9]+.[0-9]+.x"
+    tags:
+    - 'v[0-9]+.[0-9]+.[0-9]+*'
 
 env:
-  REGISTRY: nvcr.io/nvstaging/mellanox  # for docker
-  IMAGE_NAME: network-operator
-  VALID_TAG_REGEX: 'v[0-9]+\.[0-9]+\.[0-9]+(-(beta|rc)\.[0-9]+)?'
-  NGC_REPO: nvstaging/mellanox/network-operator  # for helm
+  STABLE_BRANCH: master
```

Comment:
Not the stale branch, probably, maybe just `MAINLINE_BRANCH`/`CD_BRANCH` or smth like this?

Reply:
you're right, we develop there so it's _not_ stable. i'll call it default - since that's the term used also in the github repository settings.